csrc/quantization/utils.cuh

```diff
@@ -0,0 +1,37 @@
+#pragma once
+
+/**
+ * quantization utilities.
+ */
+
+#include <cmath>
+#include <torch/all.h>
```

Comment:
I don't think `all.h` is necessary here, could you get rid of it and only include the necessary headers?